src/main/java/pl/edu/mimuw/Variable.java

```diff
@@ -0,0 +1,37 @@
+package pl.edu.mimuw;
+
+public class Variable extends MathExpression {
+  public static Double assignedValue = null;
```

Comment:
czemu to jest statyczne pole a nie klasowe? może działanie z klasowym polem wymagałoby co prawda wiecej przekazywania jakiejs zmiennej `Variable` ale pozwoliloby na posiadanie wielu instancji tej klasy